packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3635,6 +3630,23 @@ class _ExitWidgetSelectionButtonGroupState extends State<_ExitWidgetSelectionBut
 
   void _exitWidgetSelectionMode() {
     WidgetInspectorService.instance._changeWidgetSelectionMode(false);
+    // Reset to default selection on tap behavior on exit.
+    _changeSelectionOnTapMode(selectionOnTapEnabled: true);
```

Comment:
nit: can we add a constant `_defaultSelectionOnTapEnabled = true` and then use the const here? That way in case we need to use the default value anywhere else we can use the const

Reply:
yes, good idea